src/index.html

```diff
@@ -3,9 +3,26 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Hello Front-End</title>
+  <title>PhotoSlideShow</title>
+
+  <script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
+  <script src="ajax.js"></script>
 </head>
+
+
 <body>
-  <h1>Hello Front-End</h1>
-</body>
+  <h1>Photo Slide Show</h1>
+
+<form id="search-form">
+
+    <input id="qurey" placeholder="kittens">
```

Comment:
This id is spelled wrong, and doesn't match the ID you're selecting in your script.